mycore-base/src/main/java/org/mycore/common/xml/MCRXMLFunctions.java

```diff
@@ -856,6 +858,20 @@ public static String stripHtml(final String s) {
         return StringEscapeUtils.unescapeHtml4(res.toString()).replaceAll(TAG_SELF_CLOSING, "");
     }
 
+    /**
+     * Returns approximately the usable space in the MCR.datadir in bytes as in {@link FileStore#getUsableSpace()}.
+     *
+     * @return approximately the usable space in the MCR.datadir
+     * @throws IOException
+     * */
+    public static long getUsableSpace() throws IOException{
+        Path dataDir = Paths.get(MCRConfiguration2.getString("MCR.datadir").get());
```

Comment:
`MCR.datadir` should always be set in an ordinary MyCoRe instance. Still, should use `MCRConfiguration2.getStringOrThrow("MCR.datadir")` instead of `MCRConfiguration2.getString("MCR.datadir").get()` in order to generate a meaningful error message in the unusual case where no value is set.